src/Orchard.Web/Modules/Orchard.ContentPicker/Views/RecentContentTab.cshtml

```diff
@@ -1,5 +1,7 @@
 ﻿@using Orchard.Core.Contents.ViewModels;
 @{
+    Script.Include("RecentContentTab.js");

```

Comment:
wow, First yes,  Second, wow.  Been completely overloaded. Looking into
this now.

On Fri, Sep 25, 2015 at 7:26 AM, Dan Oliver <notifications@github.com>
wrote:

> In
> src/Orchard.Web/Modules/Orchard.ContentPicker/Views/RecentContentTab.cshtml
> <https://github.com/OrchardCMS/Orchard/pull/5844#discussion_r40423913>:
>
> > @@ -1,5 +1,7 @@
> >  ﻿@using Orchard.Core.Contents.ViewModels;
> >  @{
> > +    Script.Include("RecentContentTab.js");
>
> This file is in the resource manifest so should this be
> Script.Require("RecentContentTab");?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OrchardCMS/Orchard/pull/5844/files#r40423913>.
>